libs/designsystem/src/lib/components/calendar/calendar.component.scss

```diff
@@ -91,13 +95,15 @@ td {
 }
 
 .day {
+  display: inline-block;
```

Comment:
Since `.day` is applied to `<div>`s and there's ever only one of them in a cell you don't need to declare this. The `<div>` already has block display and the centered text alignment is declared below.

Another option to consider is to center using Flexbox. Then can remove the `line-height` and `text-align` declarations. If you use `inline-flex` the horizontal centering is controlled by `text-align: center` on the table cells.

Reply:
Really good suggestion, aligning items with `inline-flex` also seems to get rid of a very slight mis-alignment that each number currently has within the `.day` `<div>`.

Reply:
Implemented in 6e10812f252fb47ce917ea413a5d2d1a30722f67.
Ended up removing the text align altogether, to simplify a bit, so everything is controlled via flexbox.